pyproject.toml

```diff
@@ -22,3 +22,41 @@ exclude = '''
 )/
 '''
 
+
+[tool.check-manifest]
+ignore = [
+    ".bumpversion.cfg",
+    "CHANGES/**",
+    "CONTRIBUTING.rst",
+    "HISTORY.rst",
+    "dev_requirements.txt",
+    "doc_requirements.txt",
+    "functest_requirements.txt",
+    "unittest_requirements.txt",
+    "requirements/**",
```

Comment:
maybe this can be helpful

Reply:
we have lots of requirements on that folder and they are specific to the docker environment `standalone|insights` I think it can be kept out and the py package carries only what is defined on setup.py

Reply:
I included only the requirements/requirements.common.txt on the MANIFEST